subscriptions/forms.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+from django import forms
+from django.conf import settings
+
+
+def describe_plan(i):
+    s = settings.PRICING[i]
+    if s['calls'] == '0':
+        desc = '£%d/mth - Unlimited calls' % s['price']
+    else:
+        desc = '£%d/mth - %s calls per month' % (s['price'], s['calls'])
+    return (s['plan'], desc)
+
+
+class SubscriptionMixin(forms.Form):
+    plan = forms.ChoiceField(choices=(
+        describe_plan(i) for i in range(3)
+    ), label='Please choose a plan', widget=forms.RadioSelect)
+    charitable_tick = forms.BooleanField(
+        label='I qualify for a charitable discounted price',
+        required=False)
+    charitable = forms.ChoiceField(required=False, choices=(
```

Comment:
I can't either! No idea what I did to get it in the first place. Suspect it was something to do with having changed from a charitable to non-charitable plan.